bolts/fair/data/datamodules/tabular/admissions.py

```diff
@@ -0,0 +1,54 @@
+"""Admissions Dataset."""
+from typing import Optional
+
+import ethicml as em
+from ethicml.data.tabular_data.admissions import AdmissionsSplits
+from ethicml.preprocessing.scaling import ScalerType
+from kit import parsable
+from kit.torch import TrainingMode
+
+from .base import TabularDataModule
+
+__all__ = ["AdmissionsDataModule"]
+
+
+class AdmissionsDataModule(TabularDataModule):
+    """Data Module for the Admissions Dataset."""
+
+    @parsable
+    def __init__(
+        self,
+        sens_feat: AdmissionsSplits = "Gender",
+        disc_feats_only: bool = False,
+        # Below are super vars. Not doing *args **kwargs due to this being parsable
```

Comment:
this is one annoyance that dataclasses solve, but we probably can't combine DataModules and dataclasses